valhalla/jawn/src/lib/stores/LogStore.ts

```diff
@@ -145,6 +145,32 @@ export class LogStore {
         }
       });
 
+      try {
```

Comment:
This doesn't need another await db.tx. It should be in the one above.

We just need to get the queries and batch them